src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -652,9 +652,10 @@ public void closeBrowser() throws FixtureException {
     public void waitUntilElementFound(String elementLocator, LocatorStrategy locatorType, int timeOutInSeconds)
             throws FixtureException {
         WebDriverWait webDriverWait = new WebDriverWait(driver, timeOutInSeconds);
-        logger.trace("Timeout for WebDriverWait is set to {}.", timeOutInSeconds);
+        logger.debug("Timeout for WebDriverWait is set to {}.", timeOutInSeconds);
```

Comment:
keep trace, put a debug on top of the method "waiting max {} seconds until element {} with strategy {} is found"